pom.xml

```diff
@@ -77,6 +62,11 @@
       <artifactId>maven-scm-provider-svnjava</artifactId>
       <version>1.12</version>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.lib</groupId>
+      <artifactId>support-log-formatter</artifactId>
+      <version>1.2</version>
+    </dependency>
```

Comment:
Using our standard project-wide JUL formatter for consistency with Jenkins core and other Jenkins components.